doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -376,6 +376,10 @@ Other libraries
     The library is no longer directly referenced from the Event Manager.
     Instead, it uses the Event Manager hooks to connect with the manager.
 
+* :ref:`nrf_modem_lib_readme` library:
+
+  * Added :kconfig:option:`CONFIG_NRF_MODEM_LIB_LOG_FW_VERSION_UUID` to enable logging for both FW version and UUID at the end of the library initialization step.
+
```

Comment:
This should be below Modem libraries (Modem libraries heading is on line number 275).
Please add this statements after line number 299.